src/black/lines.py

```diff
@@ -448,7 +448,14 @@ def _maybe_empty_lines(self, current_line: Line) -> Tuple[int, int]:
         depth = current_line.depth
         while self.previous_defs and self.previous_defs[-1] >= depth:
             if self.is_pyi:
-                before = 0 if depth else 1
+                assert self.previous_line is not None
```

Comment:
Is this necessary given `self.previous_defs`?

Reply:
I'm guessing this was added to please mypy.